test/test_view_ops.py

```diff
@@ -806,7 +808,8 @@ def assert_is_nonview(t, nv):
             idx_nv = (0,) * nv.ndim
             self.assertTrue(not nv._is_view())
             nv[idx_nv] = 0
-            self.assertNotEqual(t[idx_t], nv[idx_nv])
+            if device != "meta":
+                self.assertNotEqual(t[idx_t], nv[idx_nv])
```

Comment:
A thought: we could introduce a more specific "assertDataNotEqual" which on meta tensors would assert that the metadata is equal and otherwise pass.

Reply:
That could work if we restrict the inputs to tensors. Otherwise we are going to have a hard time dissecting containers such as lists or dicts to find meta tensors without duplicating the functionality of `assertEqual`.

Reply:
That seems perfectly fine to me.

Reply:
I'll send a follow-up.